interface/src/app/analyze/analyze.js

```diff
@@ -52,6 +53,21 @@ angular.module('adage.analyze', [
         return classList;
       },
 
+      item_tooltip: function(search_item) {
+        // Determine what text to show as a tooltip for this search_item.
+        if (SampleBin.hasItem(search_item)) {
+          return 'already added to analysis';
```

Comment:
Good question. I guess we should have a consistent style for that. I'm thinking that because it's a "tip" (and not a complete sentence) the capitalization is not needed. What do you think?